client/src/legacy/UploadField_downloadtemplate.js

```diff
@@ -1,4 +1,4 @@
-window.tmpl.cache['ss-uploadfield-downloadtemplate'] = tmpl(
+tmpl.cache['ss-uploadfield-downloadtemplate'] = tmpl(
```

Comment:
Could add `window.tmpl` like in `UploadField_uploadtemplate.js` for consistency, not a breaking change